books/RayTracingInOneWeekend.html

```diff
@@ -3113,6 +3120,21 @@
 
 </div>
 
+Now we'll update the `sphere` constructor to take a material.
```

Comment:
Nit: "Now we'll update the `sphere` constructor to take a material pointer." might be more inline with the `TODO` comment.

Reply:
Done.